tests/mocks/get_user_repos_response.json

```diff
@@ -0,0 +1,55 @@
+[ {
```

Comment:
I'd slightly reformat this json to match the output of `cat tests/mocks/get_user_repos_response.json | jq .`

Reply:
thanks. updated.